Assets/Scripts/pickupGem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class pickupGem : MonoBehaviour
```

Comment:
Classes should be capitalized, remake this class and remove Functions that aren't used